### ISSUE TITLE
CloudWatch DescribeMetrics has empty value for ActionsEnabled

### ISSUE THREAD

**Issue by etwillbefine** [localstack/localstack#2161]
When creating a metrics alarm with Terraform using the terraform-aws provider  the request fails with the following:

```
 2020-03-16T16:51:47.277Z [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/16 16:51:47 [DEBUG] [aws-sdk-go] <DescribeAlarmsResponse xmlns="http://monitoring.amazonaws.com/doc/2010-08-01/"><DescribeAlarmsResult><MetricAlarms><member><ActionsEnabled></ActionsEnabled><AlarmActions><member>arn:aws:sns:eu-west-1:000000000000:org-testing-alarming</member></AlarmActions><AlarmArn></AlarmArn><AlarmConfigurationUpdatedTimestamp>2020-03-16 16:49:45.477421</AlarmConfigurationUpdatedTimestamp><AlarmDescription>Error during AWS Lambda function invocation org-testing-lambda-with-metrics</AlarmDescription><AlarmName>org-testing-lambda-with-metrics-alarm-errors</AlarmName><ComparisonOperator>GreaterThanOrEqualToThreshold</ComparisonOperator><Dimensions><member><Name>FunctionName</Name><Value>org-testing-lambda-with-metrics</Value></member></Dimensions><EvaluationPeriods>1</EvaluationPeriods><InsufficientDataActions></InsufficientDataActions><MetricName>Errors</MetricName><Namespace>AWS/Lambda</Namespace><OKActions><member>arn:aws:sns:eu-west-1:000000000000:org-testing-alarming</member></OKActions><Period>60</Period><StateReason></StateReason><StateReasonData>{}</StateReasonData><StateUpdatedTimestamp>2020-03-16 16:49:45.477427</StateUpdatedTimestamp><StateValue>OK</StateValue><Statistic>Maximum</Statistic><Threshold>1</Threshold><Unit>Count</Unit></member></MetricAlarms></DescribeAlarmsResult></DescribeAlarmsResponse>
 2020-03-16T16:51:47.277Z [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/16 16:51:47 [DEBUG] [aws-sdk-go] DEBUG: Unmarshal Response monitoring/DescribeAlarms failed, attempt 9/25, error SerializationError: failed decoding Query response
 2020-03-16T16:51:47.277Z [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 	status code: 200, request id: WC2W7BC90VAQBR4O19ILPNPZIV2L5GZURP4S6Q47ITK62UCNSNHU
 2020-03-16T16:51:47.278Z [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: caused by: strconv.ParseBool: parsing "": invalid syntax
```

The value in `ActionsEnabled` is empty. Which should convert to false according to go parse? 

When I use the AWS CLI to create a MetricAlarm in AWS using the following:
`aws cloudwatch put-metric-alarm --alarm-name test-alarm-metric --evaluation-periods 1 --comparison-operator GreaterThanOrEqualToThreshold --namespace AWS/Lambda --threshold 1 --metric-name Errors --period 60 --statistic Maximum` 

I get the following from AWS on `describe-alarms`:

```json
{
    "MetricAlarms": [
        {
            "EvaluationPeriods": 1,
            "AlarmArn": "arn:aws:cloudwatch:eu-central-1:390233841206:alarm:test-alarm-metric",
            "StateUpdatedTimestamp": "2020-03-16T17:16:56.060Z",
            "AlarmConfigurationUpdatedTimestamp": "2020-03-16T17:16:56.060Z",
            "ComparisonOperator": "GreaterThanOrEqualToThreshold",
            "AlarmActions": [],
            "Namespace": "AWS/Lambda",
            "Period": 60,
            "StateValue": "INSUFFICIENT_DATA",
            "Threshold": 1.0,
            "AlarmName": "test-alarm-metric",
            "Dimensions": [],
            "Statistic": "Maximum",
            "StateReason": "Unchecked: Initial alarm creation",
            "InsufficientDataActions": [],
            "OKActions": [],
            "ActionsEnabled": true,
            "MetricName": "Errors"
        }
]
```

when running the same against localstack using `awslocal` I get the following: 

```json
{
    "MetricAlarms": [
        {
            "AlarmName": "test-alarm-metric",
            "AlarmArn": "",
            "AlarmDescription": "None",
            "AlarmConfigurationUpdatedTimestamp": "2020-03-16 17:20:38.438675",
            "ActionsEnabled": false,
            "OKActions": [],
            "AlarmActions": [],
            "InsufficientDataActions": [],
            "StateValue": "OK",
            "StateReason": "",
            "StateReasonData": "{}",
            "StateUpdatedTimestamp": "2020-03-16 17:20:38.438680",
            "MetricName": "Errors",
            "Namespace": "AWS/Lambda",
            "Statistic": "Maximum",
            "Dimensions": [],
            "Period": 60,
            "Unit": "None",
            "EvaluationPeriods": 1,
            "Threshold": 1.0,
            "ComparisonOperator": "GreaterThanOrEqualToThreshold"
        }
    ]
}
```

The noteable difference here is that AWS sets `ActionsEnabled` to true. While localstack does not seem to set ActionsEnabled at all? Thats why it evaluates to `false` when using the CLI, but fails when using with terraform. Even when I pass `--actions-enabled` to awslocal it still reports `false` when describing the alarm.

**Comment by etwillbefine** [comment]
This is not yet implemented in https://github.com/spulec/moto/blob/6a887e52ccb73fa9e9ab21955a7c6286a82d2bdc/moto/cloudwatch/responses.py#L35 

Will try to address it there to get this into localstack.

**Comment by etwillbefine** [comment]
Will be fixed by using moto version `1.3.15.dev526` or above

**Comment by whummer** [comment]
Thanks for reporting and fixing this issue upstream @etwillbefine . The upstream changes have been merged into our fork of moto and should be available in the `latest` version of the `localstack/localstack` Docker image. Thanks
